packages/drivers/routerlicious-driver/src/wholeSummaryDocumentStorageService.ts

```diff
@@ -64,7 +65,7 @@ export class WholeSummaryDocumentStorageService implements IDocumentStorageServi
         // Fetch latest summary, cache it, and return its id.
         if (this.firstVersionsCall && count === 1) {
             this.firstVersionsCall = false;
-            const { id: _id, snapshotTree } = await this.fetchAndCacheSnapshotTree(latestSnapshotId);
+            const { id: _id, snapshotTree } = await this.fetchAndCacheSnapshotTree(latestSnapshotId, true);
```

Comment:
I think this should only be `true` if `enableDiscovery === true`, right? If `enableDiscovery === false` then we can allow caching just like before?

Reply:
Add the check. :)